prov/gni/src/ccan/container_of.h

```diff
@@ -0,0 +1,66 @@
+#ifndef CCAN_CONTAINER_OF_H
```

Comment:
I'll get a recent copy of ccan and add in the copyright.  I think its in a
separate file.


2015-01-29 11:06 GMT-07:00 Sung-Eun Choi <notifications@github.com>:

> In prov/gni/src/ccan/container_of.h
> <https://github.com/ofi-cray/libfabric/pull/3#discussion_r23788218>:
>
> > @@ -0,0 +1,66 @@
> > +#ifndef CCAN_CONTAINER_OF_H
>
> Should original copyright info be retains in this file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ofi-cray/libfabric/pull/3/files#r23788218>.
>